elasticmock/fake_elasticsearch.py

```diff
@@ -234,7 +234,7 @@ def _normalize_index_to_list(self, index):
         # Ensure to have a list of index
         if index is None:
             searchable_indexes = self.__documents_dict.keys()
-        elif isinstance(index, str) or isinstance(index, unicode):
+        elif isinstance(index, str):
```

Comment:
Why remove the unicode check?  If we want to merge this upstream we should minimize the changes.

Reply:
`unicode` is a Python 2 type. not needed (and doesn't compile) for Python 3.